models/_sector/nft/trades/chains/solana/old/magiceden_solana_events.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(
     schema = 'magiceden_solana',
     alias = 'events',
-    
+    tags =['prod_exclude'],
```

Comment:
if we're just moving the lineage around, is there a reason this is explicitly excluded?

Reply:
oh boy, maybe CI can't handle it